src/ansys/templates/python/osl_solution/{{cookiecutter.__project_name_slug}}/.github/workflows/ci.yml

```diff
@@ -34,20 +32,19 @@ jobs:
     runs-on: [ubuntu-latest]
     steps:
       - name: Check copyright
-        uses: Solution-Applications/actions/check-copyright@v4.0.0
+        uses: Solution-Applications/actions/check-copyright@v6
         with:
-          gh-token: {{ '${{ secrets.WORKFLOW_TOKEN ' }}
+          gh-token: {{ '${{ secrets.WORKFLOW_TOKEN }}' }}
```

Comment:
is it normal the there are two sets of brackets? Shouldn't it be something like this:
 gh-token: ${{ secrets.WORKFLOW_TOKEN }}

Reply:
Yes it is normal. the inner brackets corresponds to the GH yaml syntax. And the outer ones are needed to avoid conflicts with cookiecutter syntax.